### DISCUSSION TITLE
Embedding a JBrowse 2 track in another page

### DISCUSSION THREAD

**Discussion by jdepons** [GMOD/jbrowse-components#4150] [General]
I'd like to embed a track in a gene report page.   I've crafted the following url to get to the track, but am wondering if there are additional url parameters to hide the title bar, search, bar etc.  I'd really like to just embed the track itself in an IFrame or Div.   I realize there are instructions on embedding via a React component, but we do not currently use React.

Here is my URL.  Id like to just return the track box.

https://rgd.mcw.edu/jbrowse2/index.html?config=config.json&assembly=mRatBN7.2&&loc=chr1:6000000-7000000&tracks[=](https://pipelines.rgd.mcw.edu/jbrowse2/index.html?config=config.json&assembly=mRatBN7.2&&loc=chr1:6000000-7000000&tracks=Rat mRatBN7.2 (rn7) Genes and Transcripts-mRatBN7.2

**Comment by cmdcolin** [comment]
currently there are not any options easily available to do that. we could consider adding them though, I can see how it can make an easy minimal embedded widget!

**Reply by cmdcolin** [reply]
@jdepons In JBrowse v2.10.2 we added the ability to set "&nav=false" in the URL bar so you can hide the nav bar of the linear genome view. 

I realize it doesn't actually hide the main menu bar of the app (e.g. the File, View, Help, etc)

alternatively, you can look into how to embed just the jbrowse linear genome view, which this tutorial covers https://jbrowse.org/jb2/docs/tutorials/embed_linear_genome_view/01_introduction/ doesn't require any build setup or react necessarily
